mne/forward/_lead_dots.py

```diff
@@ -253,9 +297,69 @@ def _do_self_dots_subset(intrad, rmags, rlens, cosmags, ws, volume, lut,
     return products
 
 
+def _do_cross_dots(intrad, volume, coils1, coils2, r0, ch_type,
+                   lut, n_fact):
```

Comment:
and by that, I mean a filename preferably. I see you have `mne_map_data` below, I assume that doesn't mean `mne_map_data.c` but the `mne_map_data` source code. I assume this actually lives in the mne-libs/forward somewhere...? If it is in fact `mne_map_data.c`, just add the file extension below.